README.md

```diff
@@ -161,157 +215,60 @@ with tf.Session() as sess:
 - **[Try in a colab →](http://wandb.me/tf-colab)**
 - [Docs](https://docs.wandb.com/library/integrations/tensorflow)
 
+</details>
 
-## 💨 fastai
-Visualize, compare, and iterate on fastai models using Weights & Biases with the `WandbCallback`.
-```python
-import wandb
-from fastai.callback.wandb import WandbCallback
-
-# 1. Start a new run
-wandb.init(project="gpt-3")
-
-# 2. Automatically log model metrics
-learn.fit(..., cbs=WandbCallback())
-```
-
-- **[Try in a colab →](http://wandb.me/fastai-colab)**
-- [Docs](https://docs.wandb.com/library/integrations/fastai)
+<details>
+<summary>⚡️ PyTorch Lightning</summary>
 
-
-## ⚡️ PyTorch Lightning
 Build scalable, structured, high-performance PyTorch models with Lightning and log them with W&B.
+
 ```python
 from pytorch_lightning.loggers import WandbLogger
 from pytorch_lightning import Trainer
 
-wandb_logger = WandbLogger(project="gpt-3")
+wandb_logger = WandbLogger(project="gpt-3.5")
 trainer = Trainer(logger=wandb_logger)
 ```
 
 - **[Try in a colab →](http://wandb.me/lightning)**
 - [Docs](https://docs.wandb.ai/guides/integrations/lightning)
 
+</details>
 
-## 🤗 HuggingFace
-Just run a script using HuggingFace's Trainer passing `--report_to wandb` to it
-in an environment where `wandb` is installed, and we'll automatically log losses,
-evaluation metrics, model topology, and gradients:
+&nbsp;
 
-```shell
-# 1. Install the wandb library
-pip install wandb
+# Tutorials
```

Comment:
Which Tutorials do we want to highlight here considering our target audience?

Reply:
@morganmcg1 could you provide a list please?

Reply:
@morganmcg1 Do you have a list? If not, we can omit this part for now and add it later.